src/ServiceManagerBase.sol

```diff
@@ -36,11 +39,13 @@ abstract contract ServiceManagerBase is IServiceManager, OwnableUpgradeable {
     constructor(
         IAVSDirectory __avsDirectory,
         IRegistryCoordinator __registryCoordinator,
-        IStakeRegistry __stakeRegistry
+        IStakeRegistry __stakeRegistry,
+        IPaymentCoordinator ___paymentCoordinator
```

Comment:
is IPaymentCoordinator an eigenlayer core contrac? if so, can I be autistic and ask to move it above registryCoordinator, to separate the core contracts from middleware contracts (just easier to think I find..)